### PR TITLE
Fixed chat cli rendering for relay payments (pending and canceled)

### PR DESCRIPTION
**Pull request by zapu**
Before:
```
» kb chat read tuser130160
Found PRIVATE CHAT conversation: tuser130160,tuser52115b

[1]   [tuser52115b 2h]   sent Lumens worth 15 USD (102.6829358 XLM)                                      
[2]  [tuser52115b 55m]   hi                                                                              
[3]  [tuser52115b 28m]   error sending payment: Waiting for the recipient to open the app to claim, or th
                         e sender to cancel.                                                             
```

After:
```
» kb chat read tuser130160
Found PRIVATE CHAT conversation: tuser130160,tuser52115b

[1]   [tuser52115b 2h]   canceled sending Lumens worth 15 USD (102.6829358 XLM)                          
[2]  [tuser52115b 56m]   hi                                                                              
[3]  [tuser52115b 28m]   sent Lumens worth 2 USD (13.6245264 XLM)                                        
                         Waiting for the recipient to open the app to claim, or the sender to cancel.    
```

Payment in `[1]` is canceled, payment in `[3]` is pending.

### CODE REVIEW

**Comment by patrickxb** [comment]
This looks fine to me, but @mlsteele should confirm the protocol change is ok.

File: go/client/chat_cli_rendering.go [hunk 1]
```diff
@@ -515,14 +515,18 @@ func formatSendPaymentMessage(g *libkb.GlobalContext, body chat1.MessageSendPaym
...
-
-	verb := "sent"
-	if ls == "pending" {
+	var verb string
+	statusStr := strings.ToLower(details.Status)
+	switch statusStr {
+	case "completed", "canceled", "claimable":
+		verb = "sent"
+		if details.Yanked {
+			verb = "canceled sending"
```
Comment (mlsteele) [inline go/client/chat_cli_rendering.go]: the verb for yanked is canceled but the verb for canceled is sent?

Comment (zapu) [reply go/client/chat_cli_rendering.go]: i don't think "canceled" gets plumbed there, hence the need for another field in struct. It's also "completed" when it's yanked, at least from hwat I observed

Comment (mlsteele) [reply go/client/chat_cli_rendering.go]: Ok. Everything about the CLI transform and CLI avdl types is for the CLI and not shared

**Comment by zapu** [comment]
Can you look again? I changed it to use `status` field instead of adding new field.

**Pull request merged**